src/plugins/enum/README.md

```diff
@@ -0,0 +1,19 @@
+- infos = Information about the enum plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/placements = presetstorage
+- infos/description =
+
+## Introduction ##
+
+The Enum plugin checks string values of Keys by comparing it against a list of valid values.
+
+## Usage ##
+
+The plugin checks every Key in the Keyset for the Metakey `check/enum` containing a list with the syntax `['string1', 'string2', 'string3', ..., 'stringN']` and compares each value with the string value of the Key. If no match is found an error is returned.
```

Comment:
And more about the syntax: how is it possible to use `'` within the string?

One way would be escaping with `\`, another one would be to provide different quotes (e.g. `'` and`"`) and automatically concatinate them, e.g.:
`['"' "str'ing" '"', "abc"]` would be `"str'ing"` and `abc`. Then the usage of `,` to separate would pay off. What do you think?